src/traits/networking/centralized_web_server_network.rs

```diff
@@ -285,6 +289,9 @@ impl<
                 vote_index = 0;
             }
         }
+        Err(NetworkError::CentralizedWebServer {
+            source: CentralizedWebServerNetworkError::ClientDisconnected,
+        })
```

Comment:
If this loop stops running, does that mean the Client is disconnected?  Wondering what the Client is in this case and what the difference is between this and the replica being shutdown (e.g. `NetworkError::ShutDown`)

Reply:
Hmm good question- the only way for the polling loop to exit is for the `inner.running` boolean to become false, which happens in the ConnectedNetwork `shutdown()` function, so you're right: `NetworkError::ShutDown` would probably make more sense here

Reply:
Ah, gotcha! 👍